server/data/data-jmap/src/main/java/org/apache/james/jmap/api/vacation/Vacation.java

```diff
@@ -100,9 +100,6 @@ public Builder copy(Vacation vacation) {
 
         public Vacation build() {
             boolean enabled = isEnabled.orElse(DEFAULT_DISABLED);
-            if (enabled) {
```

Comment:
No, I trust you, but it's definitely a good thing to put test modifications close to modifications